XRTK.SDK/Packages/com.xrtk.sdk/Features/Diagnostics/Prefabs/DefaultDiagnosticsWindow.prefab

```diff
@@ -285,8 +285,8 @@ MonoBehaviour:
       Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
   m_text: '{PackageID} v{Package Version}'
   m_isRightToLeft: 0
-  m_fontAsset: {fileID: 11400000, guid: 8f586378b4e144a9851e7b34d9b748ee, type: 2}
-  m_sharedMaterial: {fileID: 2180264, guid: 8f586378b4e144a9851e7b34d9b748ee, type: 2}
+  m_fontAsset: {fileID: 0}
+  m_sharedMaterial: {fileID: 0}
```

Comment:
Can we break out these diagnostics window changes out into a different PR to keep the changes targeted?
Thanks

Reply:
Done